docker-compose.yml

```
@@ -9,9 +9,48 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: adminpassword
     volumes:
-      - db-data:/var/lib/postgresql/data
+      - pqdb-data:/var/lib/postgresql/data
     ports:
       - "5432:5432"
 
+        ## Redis Service   
+  redis:
+    image: redis:6.2-alpine
+    restart: unless-stopped
+    container_name: redis
+    ports:
+      - "6379:6379"
+    command: redis-server --save 60 1 --requirepass MDNcVb924a --loglevel warning
```

I would remove the need for password, since this is just for local dev after all.
```suggestion
    command: redis-server --save 60 1 --loglevel warning
```